### ISSUE TITLE
Fails to set profile in newer Firefox/Geckodriver versions

### ISSUE THREAD

**Issue by dombre** [webdriverio/wdio-firefox-profile-service#12]
The profile defined in wdio.conf is not set in newer versions of firefox and geckodriver

Firefox: 56.0
geckodriver: 0.19.0
selenium-server-standalone: 3.6.0

According to https://github.com/saadtazi/firefox-profile-js/issues/95 the browser capabilities changed and the profile should be set like so: 

```
cap['moz:firefoxOptions'] = {
	profile: zippedProfile
};
```

Instead of the old definition

```
cap.firefox_profile = zippedProfile;
```

**Comment by kkgomez2** [comment]
I'm seeing this too, tried installing this add-on because I need to set configure some profile settings, it wasn't taking the profile I set in my wdio.conf.js file.

Firefox: 56.0.2
Geckodriver: 0.19.1
with selenium-standalone using 3.6.0 version of selenium.

Rolling back to 3.4.0 seemed to work though. I would like to see this work with the latest version of selenium though.

**Comment by christian-bromann** [comment]
Fixed in #14
